### ISSUE TITLE
Visual Motivation Conversion

### ISSUE THREAD

**Issue by Unthrottled** [zd-zero/waifu-motivator-plugin#132]
Now that motivation can be consumed in visual format. I think it time to convert other motivations to become visual as well.

- [ ] Test Failures

- [ ] Startup Greeting

- [ ] Exit Salutation

**Comment by zd-zero** [comment]
Can we include the `"Motivate Me" (alt + M)` action to the conversion?

**Comment by Unthrottled** [comment]
Hmm. I thought about that. I'm just trying to come up with images that fit `Motivation`. What motivates you and motivates me could be different. Just as long as its SFW and fits the theme I think we should be good! With that said, I think a lot of the pre-existing images might count as motivational. I'll go through and add '`MOTIVATION`, and you should too. 
Let's try not to collide. I'll let you go first.

I'll also add some more images and tag them as motivational as well, once you are done.

**Comment by zd-zero** [comment]
> I think a lot of the pre-existing images might count as motivational.

I agree, As long as it's cute that'll motivate me lol

**Comment by Unthrottled** [comment]
What do you think about having `Motivate Me` display an image with no title? Also what sound should be played?

**Comment by zd-zero** [comment]
Hmmm, For motivation, I prefer to play something that is common to anime sounds and basically a waifu cheering someone.

>  ... display an image with no title
We can do this as well, but I'm thinking if we can retain it, we will have consistent alerts for other events. But, without the title, it'll have a cleaner effect as it's a user request. Can you share the reason?

**Comment by Unthrottled** [comment]
> Can you share the reason?

Yeah, so what I had planned is that many category of images/reaction-gifs can be used as `Motivation`. So I think that is really difficult to come up a one size fits all message, with the current way it implemented.

I think if we could change the asset definition to 

```
  {
    "imagePath": "celebration/megumin_yes.gif",
    "imageAlt": "",
    "imageDimensions": {
      "width": 480,
      "height": 270
    },
    "primaryCategory": "CELEBRATION"
    "subcategories": [
      "MOTIVATION"
    ]
  }
```
Then we could easily support a relevant message works with the image being displayed.
